site/vue/src/main.ts

```diff
@@ -0,0 +1,29 @@
+import { createApp } from 'vue';
+import Unknown from './pages/Unknown.vue';
+
+const exports = {
+    async render(target: string, page: string, args: Record<string, unknown> = {}) {
+        const root_component = await (async () => {
+            try {
+                return (await import(`./pages/${page}.vue`) as { default: Parameters<typeof createApp>[0] }).default;
+            }
+            catch {
+                return Unknown;
```

Comment:
Should we also log something to the console?

Reply:
Addressed in https://github.com/Submitty/Submitty/pull/10715/commits/42b22c2d10a175de7e298df3442d5ec19cc8e4fc